gst/nnstreamer/tensor_query/tensor_query_serversrc.h

```diff
@@ -45,6 +45,7 @@ struct _GstTensorQueryServerSrc
   GstPushSrc element; /* parent object */
   guint src_id;
   gboolean configured;
+  gboolean is_live;
```

Comment:
IMO, this is unnecessary.
1. get-property
g_value_set_boolean (value, gst_base_src_is_live (GST_BASE_SRC (src)));
2. set-property
gst_base_src_set_live (GST_BASE_SRC (src), g_value_get_boolean (value));
3. init
gst_base_src_set_live (GST_BASE_SRC (src), DEFAULT_IS_LIVE);

Reply:
Thanks for the suggestion. Updated!